vars/mtaBuild.groovy

```diff
@@ -11,12 +11,14 @@ def call(Map parameters = [:]) {
 
     Set parameterKeys = [
         'buildTarget',
-        'mtaJarLocation'
+        'mtaJarLocation',
+        'applicationName'
     ]
 
     Set stepConfigurationKeys = [
         'buildTarget',
-        'mtaJarLocation'
+        'mtaJarLocation',
+        'applicationName'
```

Comment:
We have as some kind of (undocumentend) rule to use the alphabetical order (ascending) here.

Reply:
done